src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -737,6 +737,9 @@
   - name: --customer-admin-group-id
     type: string
     short-summary: The Object ID of an Azure Active Directory Group that memberships will get synced into the OpenShift group "osa-customer-admins". If not specified, no cluster admin access will be granted.
+  - name: --loganalytics-workspace-resource-id
```

Comment:
@troy0820 @jim-minter is there a particular reason for choosing this name?
it seems in az cli context `--workspace-resource-id` already refers to Log Analytics workspace. If we use this name instead it will be consistent with `az aks create` and `az aks enable-addons`, also will make it shorter which is in my opinion is nicer experience (even if  with autocompletion available)

Reply:
agree with @olga-mir about having consistent and precise naming

Reply:
> Azure Log Analytics Workspace has both Id (i.e. GUID) and Resource Id, I think, we should rename to "WORKSPACE_ID" to "WORKSPACE_RESOURCE_ID" . Since we have seen customers passing WorkspaceID (i.e. GUID) instead of Azure Resource Id of the Log analytics workspace.

I think the decision came from the comment @ganga1980 made earlier in this posting which @jim-minter suggested we go with the longer name.  

> would something like --loganalytics-workspace-resource-id be better here? I know it's a lot to type. Don't mind whether it's loganalytics or monitoring or some other name.

I can change this to `monitoring` or `loganalytics` which will reference the same expected value for it to work.  The concept of add-ons when drafting this PR was not a thing and still doesn't exist yet within the ARO cluster.  We didn't want to build that complexity into it as of right now.  

I agree that the `--workspace-resource-id` would work with naming consistency but we run up against the problem @ganga1980 mentioned earlier in this thread.  If we believe that customers will not pass in the WorkspaceID _(GUID)_ with it being defined as `--workspace-resource-id` I can change it back to this or change it to one of the suggestions @jim-minter mentioned above.

Reply:
@troy0820 Thanks for the explanation. Currently, it's already a little messy about the naming for the workspace. I'm fine that we have such name `--workspace-resource-id`. I just consider that we don't need `loganalytics`. In my opinion, the ideal situation should be that the command exposes `--workspace` and supports name/workspace-resource-id/workspace-id. However, I cannot find the conversion from workspace-id to workspace-resource-id.

Reply:
Changed to `--workspace-resource-id` in commit 2f77e2dd